setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: single
+title:  "Amazon EKS"
```

Comment:
Can you add a verb to this title?
"Set up a K8s v2 provider for Amazon EKS"

(But not in the nav, because it would be too long)